includes/wizards/class-reader-revenue-wizard.php

```diff
@@ -528,6 +595,19 @@ protected function fetch_all_data() {
 	public function api_get_donation_settings() {
 		$required_plugins_installed = $this->check_required_plugins_installed();
 		if ( is_wp_error( $required_plugins_installed ) ) {
+			if ( 'nrh' === get_option( NEWSPACK_READER_REVENUE_PLATFORM ) ) {
+				return rest_ensure_response(
+					[
+						'created'                 => true,
+						'suggestedAmounts'        => [ 0, 0, 0 ],
```

Comment:
Could we hardcode some suggested tiers here, maybe? Or even give users the option to set those tiers in Reader Revenue when the NRH UI is active?

Reply:
Good idea. [This change](https://github.com/Automattic/newspack-plugin/pull/726/commits/9405be3dcc67eca654f2c0ad89830f7621e3e8b1) provides defaults to the block, and [this commit](https://github.com/Automattic/newspack-blocks/pull/642/commits/4d58bd2f5fdf87f5f58b7cb92d1ef97f3f5d92a4) causes the block to set the default prices if the tiers haven't already been manually defined.

https://github.com/Automattic/newspack-blocks/pull/642/commits/4d58bd2f5fdf87f5f58b7cb92d1ef97f3f5d92a4